contracts/DaiHrd.sol

```diff
@@ -569,28 +562,26 @@ contract DaiHrd is ERC777, MakerFunctions {
 	}
 
 	// Utility Functions
-	function convertAttodaiToAttodaiHrd(uint256 attodai, uint256 rontodaiPerPot ) internal pure returns (uint256 attodaiHrd) {
+	function calculatedChi() public view returns (uint256 rontodaiPerPot) {
+		// mirrors Maker's calculation: rmul(rpow(dsr, now - rho, ONE), chi);
+		return safeMul27(rpow(pot.dsr(), now - pot.rho(), 10 ** 27), pot.chi());
+	}
+
+	function convertAttodaiToAttodaiHrd(uint256 attodai, uint256 rontodaiPerPot ) private pure returns (uint256 attodaiHrd) {
 		// + 1 is to compensate rounding? since attodaiHrd is rounded down
 		return attodai.mul(10 ** 27).add(rontodaiPerPot - 1).div(rontodaiPerPot);
 	}
 
-	function convertAttodaiHrdToAttodai(uint256 attodaiHrd, uint256 rontodaiPerPot ) internal pure returns (uint256 attodai) {
+	function convertAttodaiHrdToAttodai(uint256 attodaiHrd, uint256 rontodaiPerPot ) private pure returns (uint256 attodai) {
 		return attodaiHrd.mul(rontodaiPerPot).div(10 ** 27);
 	}
 
-	function calculatedChi() public view returns (uint256 rontodaiPerPot) {
-		return safeMul27(rpow(pot.dsr(), now - pot.rho(), ONE), pot.chi());
-	}
-
-	function updateAndFetchChi() internal returns (uint256 rontodaiPerPot) {
-		if (pot.rho() == now) {
-			return pot.chi(); // replace with drop()?? TODO
-		}
-		return pot.drip();
+	function updateAndFetchChi() private returns (uint256 rontodaiPerPot) {
+		return (pot.rho() == now) ? pot.chi() : pot.drip();
 	}
 
 	// Takes whatever vat dai has already been transferred to DaiHrd, gives to pot (DSR) and mints tokens for user
-	function depositVatDaiForAccount(address account) internal returns (uint256 attopotDeposit) {
+	function depositVatDaiForAccount(address account) private returns (uint256 attopotDeposited) {
```

Comment:
This rename is very questionable given the comments above, but it is a private method so it matters _way_ less to me.  I did want the tense right at least though, and to be consistent with the private `withdraw` below.  I'm open to changing both of these return value names.